learning/tour-of-beam/frontend/build.gradle.kts

```diff
@@ -0,0 +1,158 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+evaluationDependsOn(":playground:frontend:playground_components")
+
+tasks.register("generate") {
+  dependsOn(":playground:frontend:playground_components:generate")
+
+  dependsOn("generateCode")
+
+  group = "build"
+  description = "Generates all generated files."
+}
+
+tasks.register("analyze") {
+  dependsOn(":playground:frontend:playground_components:generateCode")
+  dependsOn("generateCode")
+
+  group = "verification"
+  description = "Analyze dart code"
+
+  doLast {
+    exec {
+      // Exact paths instead of '.' so it does not go into playground_components
+      executable("dart")
+      args("analyze", "lib", "test")
+    }
+  }
+}
+
+tasks.register("pubGet") {
+  group = "build"
+  description = "Get packages for the playground frontend project"
```

Comment:
Strip project names from descriptions so the same ones can be used for both Playground and ToB.